VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/participant-hearings/hearing-list-table.component.spec.ts

```diff
@@ -7,13 +10,24 @@ describe('HearingListTableComponent', () => {
     let component: HearingListTableComponent;
     const translateService = translateServiceSpy;
     const testData = new ConferenceTestData();
+    let mockedHearingVenueFlagsService: HearingVenueFlagsService;
```

Comment:
Better to set the type to make it clear its a spy.
```suggestion
    let mockedHearingVenueFlagsService: jasmine.SpyObj<HearingVenueFlagsService>;
```